src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskSerializer.scala

```diff
@@ -118,9 +122,16 @@ object TaskSerializer {
       status.mesosStatus.foreach(status => builder.setStatus(status))
       builder.addAllPorts(hostPorts.map(Integer.valueOf).asJava)
     }
+    def setMarathonTaskStatus(marathonTaskStatus: MarathonTaskStatus): Unit = {
+      //scalastyle:off null
+      if (marathonTaskStatus != null) {
```

Comment:
Tests...I am simulating existing Tasks without MarathonTaskStatus in the repo and unfortunately toProto is called there...